### ISSUE TITLE
Not sure about commands in SFTP deploy target

### ISSUE THREAD

**Issue by pythoninja** [mkloubert/vscode-deploy-reloaded#76]
### Description

According to wiki's [sftp page](https://github.com/mkloubert/vscode-deploy-reloaded/wiki/target_sftp) it should be easy to chmod or remove (or anything else) after upload or before upload file. I have this config.json in my workspace:

```json
{    
    "deploy.reloaded": {
        "packages": [
            {
                "name": "shadowsocks-libev",
                "description": "Uploads shadowsocks-libev.sh",

                "files": [
                    "**/shadowsocks-libev.sh",
                ],
                "exclude": [
                ],
            }
        ],

        "targets": [
            {
                "type": "sftp",
                "name": "Vagrant",
                "description": "A SFTP folder",

                "host": "localhost", "port": 2222,
                "user": "vagrant",
                "privateKey": "/home/user/Codebase/virtual-environment/vagrant/centos/single-machine/.vagrant/machines/default/virtualbox/private_key",

                "dir": "/tmp/shadowsocks",

                "commands": {
                    "beforeUpload": [
                        "cp ${remote_file} ${remote_file}.bak"
                    ],
                    
                    "uploaded": [
                        "chmod +x ${remote_file}"
                    ]
                },
                
                "debug": true,
            },
        ]
    }
}
```

### Actual behavior

```
[root@localhost shadowsocks]# ls -la
total 20
drwxrwxr-x. 2 vagrant vagrant    34 jun 16 02:42 .
drwxrwxrwt. 9 root    root      211 jun 16 02:37 ..
-rw-rw-rw-. 1 vagrant vagrant 17939 июн 16 02:43 shadowsocks-libev.sh
```

### Expected behavior

1. chmod `+x for my shadowsocks-libev.sh` file
2. execution `cp shadowsocks-libev.sh shadowsocks-libev.sh.bak`

#### Logs

I do not understand how I can grab debug logs because of ~/.vscode-deploy-reloaded/.logs/ doesn't contains anything usefull except of:

```txt
INFO extension.deploy.reloaded.loadplugins - [16/Jun/2018:12:58:11 +0000] "Loaded 19 plugins:
- app
- azureblob
- batch
- compiler
- dropbox
- each
- ftp
- list
- local
- mail
- map
- prompt
- s3bucket
- script
- sftp
- slack
- switch
- test
- zip
"
```

### Your environment

- Operating system: Ubuntu 18.04 LTS
- Visual Studio Code version: 1.24.1
- Extension version: 0.75

**Comment by mkloubert** [comment]
@pythoninja 

**The bug should be fixed since version 0.75.1**.

All you do is correct ... "only" the [variables](https://github.com/mkloubert/vscode-deploy-reloaded/wiki/target_sftp#values) were not replaced.

**Comment by pythoninja** [comment]
@mkloubert great! Thank you for quick fix. It works as expected. I will close this issue.
